siliconcompiler/tools/openroad/openroad.py

```diff
@@ -468,22 +460,20 @@ def build_pex_corners(chip):
 
     corners = {}
     for constraint in chip.getkeys('constraint', 'timing'):
-        libcorner = chip.get('constraint', 'timing', constraint, 'libcorner',
-                             step=step, index=index)
         pexcorner = chip.get('constraint', 'timing', constraint, 'pexcorner',
                              step=step, index=index)
 
-        if not libcorner or not pexcorner:
+        if not pexcorner:
             continue
-        corners[libcorner[0]] = pexcorner
```

Comment:
Is this the fundamental reason that before only one `libcorner` was possible?

Reply:
it was a misunderstanding when this was implemented about the libcorners